test/unit/org/apache/cassandra/db/compaction/CompactionsBytemanTest.java

```diff
@@ -127,9 +128,10 @@ public void testCompactingCFCounting() throws Throwable
         cfs.enableAutoCompaction();
 
         execute("INSERT INTO %s (k, c, v) VALUES (?, ?, ?)", 0, 1, 1);
-        assertEquals(0, CompactionManager.instance.compactingCF.count(cfs));
+        Util.spinAssertEquals(true, () -> {return CompactionManager.instance.compactingCF.count(cfs) == 0;}, 1);
```

Comment:
Given in 17169 we've had timeouts on 1s tasks maybe I should do 10s here. Serves the same purpose and will avoid false flaky positives.

Reply:
Nit: can we simplify the lambda?
```suggestion
        Util.spinAssertEquals(true, () -> CompactionManager.instance.compactingCF.count(cfs) == 0, 1);
```

Reply:
Increasing to 10s or even more sounds good given our experience with timeouts. I understand that the 1s timeout matches the 1s delay injected by the Byteman rule, is this right? In that case we should also increase the Byteman delay, so the delay >= timeout.